src/components/userPage/UserPage.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import styles from './userPage.module.css';
+import { searchUser } from "../../helpers/searchUsers";
+
+class UserPage extends Component {
+  state = {
+    user: []
```

Comment:
if we are getting in user objects -we need to rename basic `user:[]` to `user: null`